commons/commons-bio/src/main/kotlin/ebi/ac/uk/model/TableModel.kt

```diff
@@ -119,4 +118,7 @@ class SectionsTable(sections: List<Section> = emptyList()) : Table<Section>(sect
         override val id = t.accNo!!
         override val attributes = t.attributes
     }
+
+    fun asSectionsTable() = SectionsTable(
```

Comment:
this shows the problem with wrongly used inheritance and the issues in our design, we do not know what instances have this collection and we need to check it manually.